docs/README.md

```diff
@@ -16,61 +16,45 @@ The LogDNA agent is a blazingly fast, resource efficient log collection client,
 
 ## Table of Contents
 
-* [Upgrading](#upgrading)
-    * [Preamble](#preamble)
-    * [Upgrading on Kubernetes](#upgrading-on-kubernetes)
-* [Installing](#installing)
-    * [Prerequisites](#prerequisites)
-    * [Installing on Kubernetes](#installing-on-kubernetes)
+* [Managing Deployments](#managing-deployments)
+  * [Installing](#installing)
+  * [Upgrading](#upgrading)
+  * [Uninstalling](#uninstalling)
+  * [More Information](#more-information)
 * [Building](#building)
-    * [Building on Linux](#building-on-linux)
-    * [Building on Docker](#building-on-docker)
+  * [Building on Linux](#building-on-linux)
+  * [Building on Docker](#building-on-docker)
 * [Configuration](#configuration)
-    * [Options](#options)
-    * [Configuring Kubernetes](#configuring-kubernetes)
-    * [Run as Non-Root](#run-as-non-root)
+  * [Options](#options)
+  * [Configuring the Environment](#configuring-the-environment)
+  * [Configuring Lookback](#configuring-lookback)
 
-## Upgrading
+## Managing Deployments
 
-### Preamble
+The agent has been tested for deployment to Kubernetes 1.9+ and OpenShift 4.6+ environments. 
```

Comment:
let's say something along the lines of "this agent supports X version"

Reply:
fixed